### PR TITLE
T-20626 - Compatible null to varchar cast for opensea

### PR DESCRIPTION
**Pull request by belen-pruvost**
Subset of changes originally added on https://github.com/duneanalytics/spellbook/pull/2029

### CODE REVIEW

File: models/opensea/opensea_events.sql [hunk 1]
```diff
@@ -55,36 +55,36 @@ FROM
...
+                CAST(NULL AS VARCHAR(5)) as tx_from,
+                CAST(NULL AS VARCHAR(5)) as tx_to,
                 NULL::double as platform_fee_amount_raw,
                 NULL::double as platform_fee_amount,
                 NULL::double as platform_fee_amount_usd,
-                NULL::string as platform_fee_percentage,
+                CAST(NULL AS VARCHAR(5)) as platform_fee_percentage,
                 NULL::double as royalty_fee_amount_raw,
                 NULL::double as royalty_fee_amount,
                 NULL::double as royalty_fee_amount_usd,
```
Comment (soispoke) [inline models/opensea/opensea_events.sql:86]: Wouldn't the `::double` fail on TrinoSQL as well?

File: models/opensea/opensea_events.sql [hunk 2]
```diff
@@ -55,36 +55,36 @@ FROM
...
-                NULL::string as royalty_fee_percentage,
+                CAST(NULL AS VARCHAR(5)) as royalty_fee_percentage,
                 NULL::double as royalty_fee_receive_address,
```
Comment (soispoke) [inline models/opensea/opensea_events.sql:88]: Same comment as above here

File: models/opensea/opensea_events.sql (hunk 1 above)
Comment (belen-pruvost) [reply models/opensea/opensea_events.sql:86]: Yes, you're right, let me fix those

**Comment by soispoke** [comment]
lgtm, let's see if this passes on prod 🤞

**Review by soispoke — APPROVED** [review]

**Pull request merged**